src/backend/controllers/users.js

```diff
@@ -95,14 +95,14 @@ export function updateProfile(req, res) {
     });
 }
 
-export async function getProfile(req, res) {
+async function getPublicProfile(req, res){
```

Comment:
@ZaiKuku told me public profile should include all the used books a user is selling. Did you and him agree that this is not needed?